fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Avatar.kt

```diff
@@ -215,6 +208,83 @@ fun Avatar(
     }
 }
 
+@Composable
+fun SlicedAvatar(
+    person: Person,
+    modifier: Modifier = Modifier,
+    width: Dp = 32.dp,
+    avatarToken: AvatarTokens? = null,
+    slicedAvatarSize: Dp = 32.dp,
```

Comment:
is slicedAvatarSize required. can we use avatarsize instead

Reply:
slicedAvatarSize would be dependent on the type of slice, for example for half it would be different and for quarter it would be different, hence required